rpc/handlers_servicer.go

```diff
@@ -0,0 +1,26 @@
+package rpc
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+func (s *rpcServer) PostV1ServicerRelay(ctx echo.Context) error {
+
+	// TODO: Move to the servicer module
```

Comment:
Any reason not to do it in this PR?

Reply:
For sure, I meant to leave it in as a TODO_IN_THIS_COMMIT. I just ran out time before adding the new `GetXXXModule` on the Bus interface to make it easy to do that.

Reply:
@Olshansk @0xBigBoss Not sure the purpose of this function but is it not a duplicate of `PostV1ClientRelay` from the RPC spec ima merge ina bit?